From 1254c7386ca04bda8406e58f29cf46079a098270 Mon Sep 17 00:00:00 2001
From: Lionir <git@thelion.website>
Date: Sun, 13 Jun 2021 19:40:16 -0400
Subject: [PATCH] Correct the date format and type

The type must be stable or development
The date format must be ISO 8601

See : https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-releases
---
 resources/linux/ghostwriter.appdata.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/resources/linux/ghostwriter.appdata.xml b/resources/linux/ghostwriter.appdata.xml
index ad116ed..d1f6a0b 100644
--- a/resources/linux/ghostwriter.appdata.xml
+++ b/resources/linux/ghostwriter.appdata.xml
@@ -28,6 +28,6 @@
   </provides>
   <content_rating type="oars-1.1"/>
   <releases>
-    <release version="2.0.1" date="20210516" type="release"/>
+    <release version="2.0.1" date="2021-05-16" type="stable"/>
   </releases>
 </component>
